src/Webapi/Dom/Webapi__Dom__Node.res

```diff
@@ -33,30 +33,33 @@ module Impl = (
   @get external textContent: T.t => string = "textContent"
   @set external setTextContent: (T.t, string) => unit = "textContent"
 
-  @send external appendChild: (T.t, Dom.node_like<'a>) => unit = "appendChild"
+  @send external appendChild: (T.t, ~child: Dom.node_like<'a>) => unit = "appendChild"
   @send external cloneNode: T.t => T.t = "cloneNode"
   @send external cloneNodeDeep: (T.t, @as(json`true`) _) => T.t = "cloneNode"
   @send
-  external compareDocumentPosition: (T.t, Dom.node_like<'a>) => int = "compareDocumentPosition" /* returns a bitmask which could also be represeneted as an enum, see https://developer.mozilla.org/en-US/docs/Web/API/Node/compareDocumentPosition */
-  @send external contains: (T.t, Dom.node_like<'a>) => bool = "contains"
+  external compareDocumentPosition: (T.t, ~other: Dom.node_like<'a>) => int = "compareDocumentPosition" /* returns a bitmask which could also be represeneted as an enum, see https://developer.mozilla.org/en-US/docs/Web/API/Node/compareDocumentPosition */
+  @send external contains: (T.t, ~child: Dom.node_like<'a>) => bool = "contains"
   @send external getRootNode: T.t => Dom.node = "getRootNode"
   @send
   external getRootNodeComposed: (T.t, @as(json`{ "composed": true }`) _) => Dom.node = "getRootNode"
   @send external hasChildNodes: T.t => bool = "hasChildNodes"
   @send
-  external insertBefore: (T.t, Dom.node_like<'a>, Dom.node_like<'b>) => Dom.node_like<'a> = "insertBefore"
+  external insertBefore: (T.t, ~new: Dom.node_like<'a>, ~before: Dom.node_like<'b>) => Dom.node_like<'a> = "insertBefore"
+  @send
+  external insertAtEnd: (T.t, ~new: Dom.node_like<'a>, @as(json`null`) _) => Dom.node_like<'a> = "insertBefore"
```

Comment:
Weird didn't know you could do that seems like the same thing as appendChild.

Reply:
Yeah I didn't know this existed either, until I was reading the specs while working on the branch. Maybe it doesn't make sense to have both bindings but who knows, maybe some developers prefer to use `parent.insertBefore(child, null)` 🤷‍♂️